x/merkledb/view.go

```diff
@@ -211,45 +213,32 @@ func (v *view) getRoot() maybe.Maybe[*node] {
 	return v.root
 }
 
-// Recalculates the node IDs for all changed nodes in the trie.
-// Cancelling [ctx] doesn't cancel calculation. It's used only for tracing.
-func (v *view) calculateNodeIDs(ctx context.Context) error {
```

Comment:
There are still a bunch of references to `calculateNodeIDs` in comments

Reply:
addressed